src/api/transactions.coffee

```diff
@@ -201,6 +201,33 @@ exports.getTransactions = ->
   catch e
     utils.logAndSetResponse this, 500, "Could not retrieve transactions via the API: #{e}", 'error'
 
+recursivelySearchObject = (ctx, obj, ws, repeat) ->
+  if Array.isArray obj
+    obj.forEach (value) ->
+      if value && typeof value is 'object'
+        if ws.has(value) then return
+        ws.add(value)
+        repeat(ctx, value, ws)
+  else
+    if obj && typeof obj is 'object'
```

Comment:
Is there any possibility that 'obj' variable could be something other than array/object? maybe a plain String or Boolean? and what would happen if it is